g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/translation/dao/impl/LocalSingleComponentDaoImpl.java

```diff
@@ -135,6 +136,7 @@ public boolean writeTranslationToFile(ComponentMessagesDTO componentMessagesDTO)
 			try {
 				FileUtils.write(targetFile, "","UTF-8",true);
 				SortJSONUtils.writeJSONObjectToJSONFile(basePath + filepath, componentMessagesDTO);
+				LOGGER.info("The bunlde file {}{} have updated sucessfully", basePath, filepath);
```

Comment:
same here

Reply:
There doesn't need to add the filepath. Because of the base path must end of filepath in configuration